views/includes/check-your-answers/beneficial-owner-statements.html

```diff
@@ -8,24 +8,82 @@
     {% endif %}
 {% endset %}
 
-<h2 class="govuk-heading-l">Has the overseas entity identified any registrable beneficial owners?</h2>
-
-{{ govukSummaryList({
-    rows: [
-      {
-        key: {
-          text: "Statement"
-        },
-        value: {
-          text: statementText
-        },
-        actions: {
-          items: [ CREATE_CHANGE_LINK(
-            OE_CONFIGS.BENEFICIAL_OWNER_STATEMENTS_URL,
-            statementText,
-            "change-beneficial-owner-statement-button"
-          ) ]
+{% set ceasedStatementText %}
+    {% if appData.update.registrable_beneficial_owner == 0 %}
+    The entity has reasonable cause to believe that at least one person has become or ceased to be a registrable beneficial owner during the update period.
+    {% elif appData.update.registrable_beneficial_owner == 1 %}
+    The entity has no reasonable cause to believe that anyone has become or ceased to be a registrable beneficial owner during the update period.
+    {% endif %}
+{% endset %}
+
+{% if pageParams.isRegistration %}
+  <h2 class="govuk-heading-l">Has the overseas entity identified any registrable beneficial owners?</h2>
+
+  {{ govukSummaryList({
+      rows: [
+        {
+          key: {
+            text: "Statement"
+          },
+          value: {
+            text: identifiedStatementText
+          },
+          actions: {
+            items: [ CREATE_CHANGE_LINK(
+              OE_CONFIGS.BENEFICIAL_OWNER_STATEMENTS_URL,
+              identifiedStatementText,
+              "change-beneficial-owner-statement-button"
+            ) ]
+          }
+        }
+      ]
+  }) }}
+
+{% else %}
+  <h2 class="govuk-heading-l">Statement details</h2>
+
+  <h2 class="govuk-heading-m">Has the overseas entity identified any registrable beneficial owners?</h2>
+
+  {{ govukSummaryList({
+      rows: [
+        {
+          key: {
+            text: "Statement"
+          },
+          value: {
+            text: identifiedStatementText
+          },
+          actions: {
+            items: [ CREATE_CHANGE_LINK(
+              OE_CONFIGS.UPDATE_BENEFICIAL_OWNER_STATEMENTS_URL,
+              identifiedStatementText,
+              "change-beneficial-owner-statement-button"
+            ) ]
+          }
+        }
+      ]
+  }) }}
+
+  <h2 class="govuk-heading-m">Have any beneficial owners become or ceased in the update period?</h2>
+
+  {{ govukSummaryList({
+      rows: [
+        {
+          key: {
+            text: "Statement"
+          },
+          value: {
+            text: ceasedStatementText
+          },
+          actions: {
+            items: [ CREATE_CHANGE_LINK(
+              OE_CONFIGS.UPDATE_REGISTRABLE_BENEFICIAL_OWNER_URL,
+              ceasedStatementText,
+              "change-beneficial-owner-statement-button"
```

Comment:
This name is same as line 60. Should it be change-registrable-beneficial-owner-button